src/Runner.jl

```diff
@@ -8,12 +8,14 @@ const AnyRedirectable = Union{Base.AbstractCmd, Base.TTY, IOStream, IOBuffer}
 # Host platform _must_ match the C++ string ABI of the binaries we get from the
 # repositories.  Note: when preferred_gcc_version=v"4" we can't really build for
 # that C++ string ABI :-(
+# Non-Linux platforms *should* end up using DockerRunner
+# so we can safely assume x86_64 support
```

Comment:
This comment doesn't make much sense.  The ability to run a x86_64 doesn't depend on using Docker.
```suggestion
```